models/installation.js

```diff
@@ -32,12 +38,10 @@ module.exports = function(Installation) {
 
   /**
    * Find installations by application id/version
-   * @param {String} deviceType The device type
+   * @param {String} deviceType The type of device (android, ios)
    * @param {String} appId The application id
    * @param {String} [appVersion] The application version
-   * @callback {Function} cb The callback function
-   * @param {Error|String} err The error object
-   * @param {Installation[]} installations The selected installations
+   * @param {Installation~findByCallback} cb The callback function passed to find()
```

Comment:
Same comment as above. Either use the convention expected by apidoc.strongloop.com, or else simplify to `@param {function(Error=,Installation[])} cb ...`